enthought/traits
# Add machinery for testing unpickling of historical pickles

This PR adds support for testing that pickled data generated with a previous version of Traits is still usable with the current version of Traits.

- add a new directory `test-data/historical-pickles`
- add pickle files for a pickled `Float` ctrait
- add a unit tests to check that historical pickles can be unpickled
- add a basic script to make it easy to generate new(!) historical pickles

I'm expecting the test to fail as a result of issue #781.
